recipes/argcomplete/bld.bat

```diff
@@ -0,0 +1,2 @@
+"%PYTHON%" setup.py install
```

Comment:
Consider replacing `bld.bat` and `build.sh` with the `script` option in the `meta.yaml`.